students/ABartles/Exercise6/test_mailroom.py

```diff
@@ -0,0 +1,67 @@
+from mailroom import *
+import os
+
+'''
+Items to Test:
+    sum_gifts
+    count_gifts
+    sorted donors
+    list_donors
+    *thanks letter
+    *send_thanks_all
+
+Items with user interaction: (can pytest test/simulate user imput?)
+    rec_donation
+    send_thanks_one
+    menu
+
+Not sure how to test w/ pytest:
+    create_report
+    run_func
+    __name__ == __main__
+'''
+
+def test_1():
+    assert sum_gifts("Anita Bath") == 350
+
+
+def test_2():
+    assert count_gifts("Seymour Butz") is 2
+
+
+def test_3():
+    name_list = ['Seymour Butz',
+                 'Anita Bath',
+                 'Isabelle Ringing',
+                 'John Smith']
+    name_list_sorted = sorted_donors()
+    for n in name_list:
+        assert n in name_list_sorted
+
+
+sample_dict = {"Tom": [100, 200], "Jane": [500]}
+
+
+def test_4():
+    assert list_donors(sample_dict) is ['Tom', 'Jane'] or ['Jane', 'Tom']
```

Comment:
this won't do what you expect because of the order of operation; if `list_donors(sample_dict) is ['Tom', 'Jane']` evaluates to False then `or ['Jane', 'Tom']` is always true because it will test for truthiness:
```
>>> assert d is  ['Tom', 'Jane'] or ['Jane', 'Tom']
>>> assert d is ['Tom', 'Jane'] or ['Jane', 'Tom']
>>> d
['cat', 'dog']
```

it sounds like you wanted to handle different order in which case you can sort your actual return:
`assert sorted(actual) == ['Jane', 'Tom']`